docs/update_and_migration/from_4.6/update_from_4.6.md

```diff
@@ -292,4 +292,25 @@ There are no additional update steps to execute.
 
 ## v4.6.18
 
-No additional steps needed.
\ No newline at end of file
+No additional steps needed.
+
+[[% include 'snippets/update/notify_support.md' %]]
+
+With the product updated to the latest version, you can now finish the update process or proceed with updating the LTS Updates packages.
```

Comment:
```suggestion
With the product updated to the latest version, you can now finish the update process or proceed to updating the LTS Updates packages.
```
I have a feeling that "proceed with" means "continue" and. "proceed to" means "go to the next task". If you agree, apply the change.